components/Ripple/examples/StatefulRippleExample.swift

```diff
@@ -49,7 +49,13 @@ class RippleView : UIView {
     statefulRippleView.isRippleHighlighted = true
   }
 
+  override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
+    statefulRippleView.superviewTouchMoved(touches.first, event: event)
```

Comment:
nit: Does it make sense to have superviewTouchMoved have the exact same signature as touchesMoved from a simplicity standpoint?

Essentially, the documentation would be:
If you have a ripple view you must call:
rippleView.touchesMoved(touches, with: event) in your implementation of touchesMoved(touches, with: event)

Reply:
Yes, I thought the same. However, if we want to leverage UIControl's `beginTrackingWithTouch`, it only takes in one UITouch as opposed to multiple. But happy to go either way. I am open to have it have the same signature. However, if we do that, we should probably have the same signatures for touchesEnded and touchesCancelled. In that case though, they get in a touch which isn't needed on our end, and also `cancelTracking` of UIControl doesn't even get a touch as an input. Just something to be aware of.

Reply:
I would match the existing signature and pass the entire set of touches.  Seems more consistent.
I would do this for all of the touches(Begin|Moved|Ended|Cancelled) methods.